Update customer-key-availability-key-roll.md (#7948)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/customer-key-availability-key-roll.md

  
## Update the keys for SharePoint Online, OneDrive for Business, and Teams files

> SharePoint Online only allows you to roll one key at a time. If you want to roll both keys in a key vault, wait for the first operation to complete. Microsoft recommends that you stagger your operations to avoid this issue. When you roll either of the Azure Key Vault keys (premium or managed HSM) associated with a DEP used with SharePoint Online and OneDrive for Business, you must update the DEP to point to the new key. This does not rotate the availability key.

Reviewer: ```suggestion
SharePoint Online only allows you to roll one key at a time. If you want to roll both keys in a key vault, wait for the first operation to complete. Microsoft recommends that you stagger your operations to avoid this issue. When you roll either of the Azure Key Vault keys (Premium or Managed HSM) associated with a DEP used with SharePoint Online and OneDrive for Business, you must update the DEP to point to the new key. This does not rotate the availability key.
```

> 1
> A. Run the Update-SPODataEncryptionPolicy cmdlet as follows (for premium key vault)

Reviewer: ```suggestion
1. Perform either one of these steps:

- **For premium key vault**: run the Update-SPODataEncryptionPolicy cmdlet as follows:
```

  
   ```powershell
   Update-SPODataEncryptionPolicy  <SPOAdminSiteUrl> -KeyVaultName <ReplacementKeyVaultName> -KeyName <ReplacementKeyName> -KeyVersion <ReplacementKeyVersion> -KeyType <Primary | Secondary>
   ```

> B. Run the Update-SPODataEncryptionPolicy cmdlet as follows (for managed HSM )

Reviewer: ```suggestion

- **For managed HSM**: run the Update-SPODataEncryptionPolicy cmdlet as follows:
```
